underlay/src/main/resources/config/display/sd/criteriaselector/bioVU/bioVU.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
nit: In the case of empty data, we could just put it inline by convention (e.g. `pluginConfig: "{}"`) instead of using a separate file.

Reply:
Done.